src/main/java/com/savvato/tribeapp/services/ConnectServiceImpl.java

```diff
@@ -112,21 +113,32 @@ public List<ConnectOutgoingMessageDTO> handleConnectionIntent(String connectionI
         if (connectionIntent == "") {
             rtn.add(ConnectOutgoingMessageDTO.builder()
                     .message("Please confirm that you wish to connect.")
-                    .to(toBeConnectedWithUserId)
+                            .to(UsernameDTO.builder()
+                                    .userId(requestingUserId)
```

Comment:
I believe when I first worked through this method, this message "Please confirm that you wish to connect." was going to the toBeConnectedWith user as a confirmation. Is this actually for the requesting user?

Reply:
Yes, thank you. Good find. I've corrected it.